release-calendar/src/client/models/release-channel.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Release} from '../../types';
+
+export function getChannelRTVs(releases: Release[]): string[] {
+  const channelRTVs: string[] = [];
+  channelRTVs.push(releases.find(release => release.channel == 'lts').name);
```

Comment:
How does `releases.find()` ensure that it gets the current release?

Reply:
I was making the assumption that when I query the releases that I can order them my date. `releases.find()` returns the first of that type, which should be the most current. Is that okay to assume that will eventually be something I can make sure of?